about.html

```diff
@@ -12,36 +12,18 @@
     We're Hackatron.
   </p>
 
-  <h2 id="where">Where</h2>
-  <div class="block">
-    <div class="venue">
-      <h3><a href="http://www.fungostudios.com/">FunGo Studios</a></h3>
-      <p>
-        Via Sile, 41<br/>
-        31056 Roncade<br/>
-        Treviso, Italy
-      </p>
-    </div>
-    <div class="venue">
-      <h3><a href="http://memecoworking.com/">Meme Coworking</a></h3>
-      <p>
-        Via Marignana, 86<br/>
-        31020 Mogliano Veneto<br/>
-        Treviso, Italy
-      </p>
-    </div>
-  </div>
-
-  <h2 id="when">When</h2>
-  <p>Every Tuesday at 6:30 pm.</p>
-
   <h2 id="irc">IRC</h2>
   <p>
     You can chat with Hackatron's team (or <a href="https://github.com/hackatron/hackatron-hubot">Hackabot</a> when no one is online) about software and life by joining the <a href="irc://irc.freenode.net/hackatron">#hackatron</a> channel at irc.freenode.net.
   </p>
 
   <iframe src="http://webchat.freenode.net/?channels=hackatron" width="100%" height="600" frameborder="0"></iframe>
 
+  <h2 id="slack">Slack</h2>
+  <p>
+    You can also find us on Slack under the <a href="https://hackatron-slackin.herokuapp.com/">Hackatron fioi</a> team.
+  </p>
```

Comment:
@potomak I do agree about Slack, but since we have that communication channel in palce, I would try to promote it anyway. It could be a good way to keep in touch with new comers anyway.